tsconfig.json

```diff
@@ -9,7 +9,8 @@
         "skipLibCheck": true,
         "forceConsistentCasingInFileNames": true,
         "declaration": true,
-        "removeComments": false,
+        "sourceMap": true,
+        "removeComments": true,
```

Comment:
this can be just fully removed, since it defaults to true

Reply:
defaults to false https://www.typescriptlang.org/tsconfig#removeComments